integrations/access/jira/app.go

```diff
@@ -290,6 +290,9 @@ func (a *App) onWatcherEvent(ctx context.Context, event types.Event) error {
 
 // onJiraWebhook processes Jira webhook and updates the status of an issue
 func (a *App) onJiraWebhook(ctx context.Context, webhook Webhook) error {
+	ctx, cancel := context.WithTimeout(ctx, webhookIssueAPIRetryTimeout)
```

Comment:
Looks like webhookIssueAPIRetryTimeout is 1 minute, should be bump it as well to, say, 5?